huffman.go

```diff
@@ -2,165 +2,43 @@ package huffman
 
 import (
 	"errors"
-	"sort"
 )
 
 const (
-	eofSymbol  = 256
```

Comment:
why did you rename those? I am not sure I like the Huffman prefix in a huffman package. Seems redundant.

Reply:
will remove the prefix